integration-test/jest/src/App.jsx

```diff
@@ -0,0 +1,81 @@
+import { Suspense, useState } from "react";
+import {
+  ApolloNextAppProvider,
+  NextSSRApolloClient,
+  NextSSRInMemoryCache,
+  useSuspenseQuery,
+} from "@apollo/experimental-nextjs-app-support/ssr";
+import { SchemaLink } from "@apollo/client/link/schema/index.js";
+import { gql, ApolloLink, Observable } from "@apollo/client/index.js";
+import { schema } from "./schema";
+
+const delayLink = new ApolloLink((operation, forward) => {
+  return new Observable((observer) => {
+    const handle = setTimeout(() => {
+      forward(operation).subscribe(observer);
+    }, 20);
+
+    return () => {
+      clearTimeout(handle);
+    };
+  });
+});
+
+export const makeClient = () => {
+  return new NextSSRApolloClient({
+    cache: new NextSSRInMemoryCache(),
+    link: delayLink.concat(new SchemaLink({ schema })),
+  });
+};
+
+function App() {
+  return (
+    <>
+      <h1>Vite + React (patched) Streaming SSR + Apollo Client + Suspense</h1>
```

Comment:
```suggestion
      <h1>Jest + Streaming SSR + Apollo Client + Suspense</h1>
```

Not sure if this is exactly right, but perhaps this should reflect that this is part of the Jest test?

Reply:
```suggestion
      <h1>Jest integration test</h1>
```